test/functional/test-cid.js

```diff
@@ -596,8 +613,7 @@ describe('cid', () => {
         cookieName: 'cookie_name',
       }, hasConsent).then(c => {
         expect(c).to.exist;
-        expect(c).to
-            .equal('amp-sha384([1,2,3,0,0,0,0,0,0,0,0,0,0,0,0,15])');
+        expect(c).to.equal('amp-AQIDAAAAAAAAAAAAAAAADw');
```

Comment:
Correct. For AMP generated cookies, we expect a particular format. The "expectation" was mostly based on the implementation at the time. If the length needs to change, I am okay with it.  For more context on what @lannka and @michaelkleber are talking about, see http://cl/154906470